metals-docs/src/main/scala/docs/TestFrameworksModifier.scala

```diff
@@ -0,0 +1,19 @@
+package docs
+
+import mdoc.StringModifier
+import mdoc.Reporter
+import scala.meta.inputs.Input
+
+class TestFrameworksModifier extends StringModifier {
+  override val name: String = "test-frameworks"
+
+  override def process(info: String, code: Input, reporter: Reporter): String =
+    """|Note: Not all test frameworks will work by default when Metals tries to
+       |detect tests for code lenses or usage in the test explorer. The current
+       |recognized test frameworks can be found below:
```

Comment:
I think that following isn't crystal clear when reading added docs:
- almost all test frameworks works out of the box with Metals wrt finding whole test classes
- for only few frameworks, Metals are able to detect single tests/test case

Maybe it's just me who don't see enough difference between test as test suite and test as single test in bigger suite 🤔

Reply:
Ah yea you're right, that's not clear. What about something like:

```
NOTE: While Metals detects test suites for most of existing testing frameworks, support for recognizing individual tests is more limited. Metals supports the current set of test frameworks when it comes to individual test discovery:

- JUnit
- MUnit
- Scalatest
```
Is there any popular frameworks that Metals doesn't work at all with?

Reply:
> Is there any popular frameworks that Metals doesn't work at all with?

I don't know of any. Hopefully that would be reported :sweat_smile: 

> Ah yea you're right, that's not clear. What about something like:

That sounds good! It would be the same as you put in the release notes, right?

Reply:
Alright, updated it in both places 👍🏼